src/lando/jinja.py

```diff
@@ -102,6 +102,58 @@ def tostatusbadgename(landing_job: LandingJob) -> str:
     return mapping.get(landing_job.status.lower(), landing_job.status.capitalize())
 
 
+def uplift_status_tag_class(job: UpliftJob) -> str:
+    try:
+        status = JobStatus(job.status)
+    except ValueError:
+        status = None
+
+    if status == JobStatus.LANDED:
+        return "is-success"
+    if status == JobStatus.FAILED:
+        return "is-danger"
+    if status == JobStatus.IN_PROGRESS:
+        return "is-info"
+    if status == JobStatus.DEFERRED:
+        return "is-warning"
+    return "is-light"
+
+
+def uplift_status_icon_class(job: UpliftJob) -> str:
+    try:
+        status = JobStatus(job.status)
+    except ValueError:
+        status = None
+
+    if status == JobStatus.LANDED:
+        return "fa fa-check"
+    if status == JobStatus.FAILED:
+        return "fa fa-times"
+    if status == JobStatus.IN_PROGRESS:
+        return "fa fa-clock-o"
+    if status == JobStatus.DEFERRED:
+        return "fa fa-history"
+    if status == JobStatus.CREATED:
+        return "fa fa-file-o"
+    if status == JobStatus.SUBMITTED:
+        return "fa fa-hourglass-start"
+    if status == JobStatus.CANCELLED:
+        return "fa fa-ban"
+    return ""
+
+
+def uplift_status_label(job: UpliftJob) -> str:
+    try:
+        status = JobStatus(job.status)
+    except ValueError:
+        return ""
+
+    if status == JobStatus.LANDED:
+        return "Uplift applied and created successfully"
```

Comment:
“applied” may be confusing, as it's not applied in the landed sort of sense yet, even though we have confirmed that it was feasible. 

```suggestion
        return "Uplift revisions created successfully"
```

Reply:
Alternatively, 
```suggestion
        return "Uplift request created successfully"
```

Reply:
Yep, looks good!